projects/js-packages/explat/composer.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "automattic/jetpack-explat",
+	"description": "JS package component and utils for A/B testing using ExPlat",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"test-js": [
+			"pnpm run test"
+		]
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
+		"autotagger": true,
+		"changelogger": {
+			"link-template": "https://github.com/Automattic/jetpack-explat/compare/v${old}...v${new}"
+		},
+		"mirror-repo": "Automattic/jetpack-explat"
```

Comment:
Since you're setting up a mirror repo for the package, do you want to set up the [Npmjs Auto-Publisher](https://github.com/Automattic/jetpack/blob/trunk/docs/monorepo.md#npmjs-auto-publisher) too?

If you're not planning on publishing it to npmjs.com, it may not be worth creating a mirror repo at all.

Reply:
I don't think we'll use it now, so I'll remove it. Also, that's something we could easily change in the future, right?

Reply:
Yes, you can always re-add the mirror-repo back here if you want to start publishing it.